spec/lib/models/i18n_repository_spec.rb

```diff
@@ -2,10 +2,11 @@
 require_relative '../../../lib/open_project/translations/models/i18n_provider'
 
 describe I18nProvider do
-  let(:i18n_provider) { described_class.new(project_id, api_key, crowdin_directory) }
+  let(:i18n_provider) { described_class.new(project_id, api_key, crowdin_directory, previous_crowdin_directory) }
   let(:project_id) { 'test-id' }
   let(:api_key) { 'test-api-key' }
   let(:crowdin_directory) { 'test-directory' }
+  let(:previous_crowdin_directory) { 'previous_directory' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.